Source/missiles.cpp

```diff
@@ -2893,50 +2947,52 @@ int AddMissile(Point src, Point dst, int midir, int mitype, int8_t micaster, int
 
 			for (int i = 0; i < ActiveMissileCount; i++) {
 				int mi = ActiveMissiles[i];
-				if (Missiles[mi]._mitype == MIS_MANASHIELD && Missiles[mi]._misource == id)
+				auto &missile = Missiles[mi];
+				if (missile._mitype == MIS_MANASHIELD && missile._misource == id)
 					return -1;
 			}
 		}
 	}
 
 	int mi = AvailableMissiles[0];
+	auto &missile = Missiles[mi];
 
 	AvailableMissiles[0] = AvailableMissiles[MAXMISSILES - ActiveMissileCount - 1];
 	ActiveMissiles[ActiveMissileCount] = mi;
 	ActiveMissileCount++;
 
 	memset(&Missiles[mi], 0, sizeof(*Missiles));

```

Comment:
```suggestion
	memset(&missile, 0, sizeof(*Missiles));

```

Reply:
@AJenbo , for this one I changed `sizeof(*Missiles)` to `sizeof(missile)` as suggested by @ephphatha in Discord development channel. 

```c++
memset(&missile, 0, sizeof(missile));
```